src/libraries/System.ComponentModel.TypeConverter/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -301,5 +301,17 @@
       <property name="Scope">member</property>
       <property name="Target">M:System.ComponentModel.Design.DesigntimeLicenseContextSerializer.Serialize(System.IO.Stream,System.String,System.ComponentModel.Design.DesigntimeLicenseContext)</property>
     </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
```

Comment:
Tagging @eerhardt. I added these here because I wanted to suppress these warnings. Not sure if this is the right thing to do and/or if there's a better method?

Reply:
Can we remove some of the suppressions that were already here for the code you refactored? The one directly above this is:

`M:System.ComponentModel.Design.DesigntimeLicenseContextSerializer.Serialize(System.IO.Stream,System.String,System.ComponentModel.Design.DesigntimeLicenseContext)`

I assume that one can be removed now.

Reply:
Oh, of course. Removed now